test/integration/stackstorm/controls/services_test.rb

```diff
@@ -27,14 +27,14 @@
   # st2auth
   describe port(9100) do
     it { should be_listening }
-    its('addresses') { should include '127.0.0.1' }
+    its('addresses') { should include '0.0.0.0' }
```

Comment:
Any reasons for changing these (and not for stream)? 

If there is nginx running, I think it should be expected to be bound to `0.0.0.0` instead, proxying for the st2{api,auth,stream} services running on `127.0.0.1`